client/clientservice/src/request_service.cpp

```diff
@@ -74,12 +74,12 @@ Status RequestServiceImpl::Send(ServerContext* context, const Request* proto_req
     bft::client::ReadConfig config;
     config.request = req_config;
     auto span = opentracing::Tracer::Global()->StartSpan("send_ro", {});
```

Comment:
Now this is an unused variable. How do we forward traces without this API?

Reply:
Added how to use bft::client::ReadConfig or 'bft::client::WriteConfig' to the function call send() in the comments.